### PR TITLE
SONARJAVA-3173 Update metadata

### PR DESCRIPTION
**Pull request by m-g-sonar**

### CODE REVIEW

File: java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1067_java.html [hunk 1]
```diff
@@ -10,4 +10,7 @@ <h2>Compliant Solution</h2>
 <pre>
 if ( (myFirstCondition() || mySecondCondition()) &amp;&amp; myLastCondition()) { ... }
 </pre>
+<h2>Exceptions</h2>
```
Comment (m-g-sonar) [inline java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1067_java.html]: Handled by [SONARJAVA-3133](https://jira.sonarsource.com/browse/SONARJAVA-3133)

File: java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2070_java.html [hunk 2]
```diff
@@ -39,5 +39,5 @@ <h2>See</h2>
   <li> <a href="http://shattered.io/">SHAttered</a> - The first concrete collision attack against SHA-1. </li>
 </ul>
 <h2>Deprecated</h2>
-<p>This rule is deprecated; use {rule:squid:S4790} instead.</p>
+<p>This rule is deprecated; use {rule:squid:S4790}, {rule:squid:S5344} instead.</p>
```
Comment (m-g-sonar) [inline java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2070_java.html]: Implemented in SonarJava 5.13 ([SONARJAVA-3104](https://jira.sonarsource.com/browse/SONARJAVA-3104))

File: java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2275_java.html [hunk 3]
```diff
@@ -36,12 +39,15 @@ <h2>Compliant Solution</h2>
...
 java.util.logging.Logger logger;
-logger.log(java.util.logging.Level.SEVERE, "Result {1}!", 14, 2); // Noncompliant {{Not enough arguments.}}
+logger.log(java.util.logging.Level.SEVERE, "Result {1},{2}!", 14, 2);
 
 org.slf4j.Logger slf4jLog;
 org.slf4j.Marker marker;
 
 slf4jLog.debug(marker, "message {}", 1);
+
+org.apache.logging.log4j.Logger log4jLog;
```
Comment (m-g-sonar) [inline java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2275_java.html]: Handled by [SONARJAVA-3172](https://jira.sonarsource.com/browse/SONARJAVA-3172)

File: java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3457_java.html [hunk 4]
```diff
@@ -52,6 +55,9 @@ <h2>Compliant Solution</h2>
 
 slf4jLog.debug(marker, "message {}");
 slf4jLog.debug(marker, "message {}", 1);
+
+org.apache.logging.log4j.Logger log4jLog;
```
Comment (m-g-sonar) [inline java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3457_java.html]: Handled by [SONARJAVA-3172](https://jira.sonarsource.com/browse/SONARJAVA-3172)

File: java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S4684_java.html [hunk 5]
```diff
@@ -75,6 +75,9 @@ <h2>Compliant Solution</h2>
   }
 }
 </pre>
+<h2>Exceptions</h2>
+<p>No issue is reported when the parameter is annotated with <code>@PathVariable</code> from Spring Framework, since the lookup will be done via id,
+the object cannot be forged on client side.</p>
```
Comment (alban-auzeill) [inline java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S4684_java.html]: Handled by SONARJAVA-3151

File: java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S4790_java.html [hunk 6]
```diff
@@ -151,3 +151,121 @@ <h2>Questionable Code Example</h2>
     }
 }
 </pre>
+<h2>Questionable Code Example</h2>
```
Comment (alban-auzeill) [inline java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S4790_java.html]: This long `Questionable Code Example` section is duplicated, rule-api should be drunk.
I'm going to change `Questionable Code Example` by `Sensitive Code Example` in the RSPEC, it should fix rule-api.

**Pull request merged**